docs/src/theme/Prerequisites.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Admonition from '@theme/Admonition';
+
+export default function Prerequisites({ titlesAndLinks }) {
+    return (
+        <Admonition type="info" title="Recommended Prereading" icon="📚">
```

Comment:
i think i like "Conceptual Prereqs" more than recommended reading — they really are prereqs for understanding what's going on, not just suggestions

Reply:
Changed back to `Prerequisites`